CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 6.17.0 (October 21, 2022). Tested on Artifactory 7.46.6
```

Comment:
Is this semantic versioning?

Just out of curiosity, how do you decide to go from 6.16.4 -> 6.17.0 vs 6.16.5

Does this PR count as a minor feature addition?

Reply:
[Terraform guidance.](https://developer.hashicorp.com/terraform/plugin/sdkv2/best-practices/versioning#versioning-specification)